config/parameters.yml

```diff
@@ -1,5 +1,5 @@
 # please update with your own information
 ---
 contact:
-  name: Example Joe
```

Comment:
Hi, I guess you pushed the commit adding this particular change to this PR only by accident. It makes perfect sense for your own installation of the API, but it won't be merged in the main `church-calendar-api` repo. (I am still grateful for the code in the first commit and will definitely merge and deploy it once I get to this task.)